recipes/ecfeed/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class HelloTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
to generate ecfeed-config.cmake

Reply:
Updated.